utils.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import githubsearch_pb2 as schema
+
+SEARCH_TERM_MAX_LEN = 256
+SEARCH_TERM_MAX_OPERATORS = 5
+
+def _check_search_validity(search_term):
```

Comment:
maybe move search_tearm_max_len and search_term_max_operators into the function signature.

Reply:
Done